src/control/ros/Conversions.cpp

```diff
@@ -429,6 +429,29 @@ trajectory_msgs::JointTrajectory toRosJointTrajectory(
   return jointTrajectory;
 }
 
+//=============================================================================
+sensor_msgs::JointState toJointState(
+  const Eigen::VectorXd& goalPositions, const std::vector<std::string>& jointNames)
+{
+  if (goalPositions.size() != jointNames.size())
+  {
+    std::stringstream message;
+    message << "The size of goalPositions ("<<goalPositions.size()<<
+      ") must be the same as jointNames ("<<jointNames.size()<<"!" ;
+    throw std::invalid_argument(message.str());
+  }
+
+  sensor_msgs::JointState jointState;
+
+  jointState.name = std::move(jointNames);
```

Comment:
Nit: It is not possible to `std::move` out of `jointNames` because it was passed by `const` reference. You should either remove the `std::move` or modify the function to take `jointNames` by value. I'd slightly prefer the latter.